docs/Changelog.rst

```diff
@@ -18,6 +18,9 @@ Changelog
     This addresses #1363, and everyone else upgrading to Elasticsearch 7.x.
     Update documentation accordingly. (untergeek)
 
+  * Prevent the ``empty`` filtertype from incorrectly matching against closed
+    indices (heyitsmdr)
```

Comment:
Please add the reference to this PR: #1430 (the other lines all refer to the issue or PR number)